git-repo-prep/static/repo-settings/.github/settings.yml

```diff
@@ -0,0 +1,45 @@
+# These settings are synced to GitHub by https://probot.github.io/apps/settings/
+
+repository:
+  # See https://docs.github.com/en/rest/reference/repos#update-a-repository for all available settings.
+  private: false
+
+  has_issues: false
+  has_wiki: false
+  has_projects: false
+  has_downloads: false
+
+  default_branch: main
+
+  allow_auto_merge: true
+  delete_branch_on_merge: true
+  allow_forking: true
+
+# Collaborators: give specific users access to this repository.
+# See https://docs.github.com/en/rest/reference/repos#add-a-repository-collaborator for available options
+collaborators:
+
+# See https://docs.github.com/en/rest/reference/teams#add-or-update-team-repository-permissions for available options
+teams:
+  - name: all
+    permission: pull
+  - name: ce-admins
+    permission: admin
+  - name: ce-all
+    permission: triage
+  - name: content-managers
+    permission: admin
```

Comment:
I thought that CM's (&ce-admin) no longer need admin permission because updating settings will be done by updating this file